NavigationReactNative/src/NavigationReactNative.ts

```diff
@@ -8,5 +8,6 @@ import SearchBarIOS from './SearchBarIOS';
 import TabBarIOS from './TabBarIOS';
 import TabBarItemIOS from './TabBarItemIOS';
 import SharedElementAndroid from './SharedElementAndroid';
+import TitleBarIOS from './TabBarIOS';
```

Comment:
Ooops! This should say
```js
import TitleBarIOS from './TitleBarIOS';
```
You can test it out by running `npm run package` from the top level navigation folder and then copying the `/build/npm/navigation-react-native` folder into the `node_modules` of the zoom sample

Reply:
OK, let me test it.